metrics/mixin/alerts/perf.libsonnet

```diff
@@ -17,6 +17,7 @@
               message: 'High MDS cache usage for the daemon {{ $labels.ceph_daemon }}.',
               description: 'MDS cache usage for the daemon {{ $labels.ceph_daemon }} has exceeded above 95% of the requested value. Increase the memory request for {{ $labels.ceph_daemon }} pod.',
               severity_level: 'error',
+              runbook_url: 'https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/CephMdsCacheUsageHigh.md',
```

Comment:
severity_level is missing

Reply:
I see it on Line 19. Is that the one you are referring to?

Reply:
only change is to add the `runbook_url` link. 
Testing looks good. 

![Screenshot from 2024-01-22 20-50-47](https://github.com/red-hat-storage/ocs-operator/assets/9363998/87c6e67e-3a64-4a6e-b20d-c88b62e40f7e)

Reply:
@umangachapagain is that how the Runbook URL link should look like in the alert?

Reply:
Yeah, I missed seeing it on Line 19

Reply:
you answered this over slack.